web/pdf_viewer.js

```diff
@@ -247,6 +251,8 @@ class PDFViewer {
       );
     }
     this.container = options.container;
+    this.#resizeObserver.observe(this.container);
```

Comment:
Is this guaranteed to always set the `--viewer-container-height` when the viewer initially loads?

(Possibly a stupid question, but I'm not all that familiar with `ResizeObserver` yet.)